.gitignore

```diff
@@ -72,3 +72,4 @@ static/src/stylesheets/pasteup/.npmrc
 *.crt
 *.key
 .vscode/settings.json
+.yarnrc
```

Comment:
Was this automatically added by an IDE?

Reply:
Sorry, just read your parenthetical! You could add this to your global gitignore if it is specific for your machine